### ISSUE TITLE
Why did you multiply 2 in your code when calculating Regulation_loss

### ISSUE THREAD

**Issue by havefunbb** [microsoft/Deep3DFaceReconstruction#147]
It's not necessary to multiply 2 in the code, if you really want to increase the Regulation_loss you could increase the loss weight in option.py. But why did you do this in your code?

**Comment by havefunbb** [comment]
`# coefficient regularization to ensure plausible 3d faces
def Regulation_loss(id_coeff,ex_coeff,tex_coeff,opt):
	w_ex = opt.w_ex
	w_tex = opt.w_tex

	regulation_loss = tf.nn.l2_loss(id_coeff) + w_ex * tf.nn.l2_loss(ex_coeff) + w_tex * tf.nn.l2_loss(tex_coeff)
	regulation_loss = regulation_loss/ tf.cast(tf.shape(id_coeff)[0],tf.float32)   # *2 ???

	return regulation_loss `

**Comment by YuDeng** [comment]
tf.nn.l2_loss actually calculates sum(x**2)/2. We multiply it with 2 in order to cancel the division inside the tf function.

**Comment by havefunbb** [comment]
OK,thanks
